google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/it/ITCommitResponseTest.java

```diff
@@ -0,0 +1,149 @@
+      connection.bufferedWrite(
+          Mutation.newInsertBuilder("TEST").set("ID").to(1L).set("NAME").to("TEST").build());
+      connection.commit();
+      assertThat(connection.getCommitResponse()).isNotNull();
+      assertThat(connection.getCommitResponse().getCommitTimestamp()).isNotNull();
+      assertThat(connection.getCommitResponse().hasCommitStats()).isTrue();
+      assertThat(connection.getCommitResponse().getCommitStats().getMutationCount()).isEqualTo(2L);
+      try (ResultSet rs =
+          connection.execute(Statement.of("SHOW VARIABLE COMMIT_RESPONSE")).getResultSet()) {
+        assertThat(rs.next()).isTrue();
+        assertThat(rs.getTimestamp("COMMIT_TIMESTAMP")).isNotNull();
+        assertThat(rs.getLong("MUTATION_COUNT")).isEqualTo(2L);
+        assertThat(rs.getString("OVERLOAD_DELAY")).isNotNull();
+        assertThat(rs.next()).isFalse();
+      }
+    }
+  }
+
+  @Test
+  public void testAutocommitDefaultNoCommitStats() {
+    try (ITConnection connection = createConnection()) {
+      connection.setAutocommit(true);
+      connection.write(
+          Mutation.newInsertBuilder("TEST").set("ID").to(1L).set("NAME").to("TEST").build());
+      assertThat(connection.getCommitResponse()).isNotNull();
+      assertThat(connection.getCommitResponse().getCommitTimestamp()).isNotNull();
+      assertThat(connection.getCommitResponse().hasCommitStats()).isFalse();
+    }
+  }
+
+  @Test
+  public void testAutocommitReturnCommitStats() {
+    try (ITConnection connection = createConnection()) {
+      connection.setAutocommit(true);
+      connection.setReturnCommitStats(true);
+      connection.write(
+          Mutation.newInsertBuilder("TEST").set("ID").to(1L).set("NAME").to("TEST").build());
+      assertThat(connection.getCommitResponse()).isNotNull();
+      assertThat(connection.getCommitResponse().getCommitTimestamp()).isNotNull();
+      assertThat(connection.getCommitResponse().hasCommitStats()).isTrue();
+      assertThat(connection.getCommitResponse().getCommitStats().getMutationCount()).isEqualTo(2L);
+    }
+  }
+
+  @Test
+  public void testAutocommitReturnCommitStatsUsingSql() {
+    try (ITConnection connection = createConnection()) {
+      connection.execute(Statement.of("SET AUTOCOMMIT=TRUE"));
+      connection.execute(Statement.of("SET RETURN_COMMIT_STATS=TRUE"));
+      connection.write(
+          Mutation.newInsertBuilder("TEST").set("ID").to(1L).set("NAME").to("TEST").build());
+      assertThat(connection.getCommitResponse()).isNotNull();
+      assertThat(connection.getCommitResponse().getCommitTimestamp()).isNotNull();
+      assertThat(connection.getCommitResponse().hasCommitStats()).isTrue();
+      assertThat(connection.getCommitResponse().getCommitStats().getMutationCount()).isEqualTo(2L);
+      try (ResultSet rs =
+          connection.execute(Statement.of("SHOW VARIABLE COMMIT_RESPONSE")).getResultSet()) {
+        assertThat(rs.next()).isTrue();
```

Comment:
in general, avoid Truth for assertions that are simpler in JUnit

Reply:
Replace all in this file.